explainer.md

```diff
@@ -24,9 +24,10 @@ A visual demo of some example transition patterns targeted by this feature are
 The following is a summary of the semantics of these transition patterns :
 
 *   Root Transitions : The full page content animates between two web pages with
-    an optional static UI element on top.
+    optional static UI elements on top.
     [Shared axis](https://material.io/design/motion/the-motion-system.html#shared-axis)
-    shows an example.
+    shows these examples. #1 and #2 have a transition with static elements while #3
```

Comment:
"I found this a little confusing at first, since what you're describing isn't 'shared axis'. Although examples 1 & 2 in the video do have a static component, example 3 doesn't."

I intentionally stated *optional* static UI elements so it was clear that shared axis patterns can be done without a static element as well.